### PR TITLE
Update managing-webhook-data.md

### PR DESCRIPTION
**Pull request by sptratledge**
As per Bob Evans's latest email after investigation with Kieran, doc updated to reflect the actual webhook batch sizing we're doing.

### CODE REVIEW

File: articles/tech-resources/managing-webhook-data.md [hunk 1]
```diff
@@ -7,7 +7,13 @@ SparkPost and SparkPost Enterprise provide data streams of raw event data via we
 
 ## Webhook Batch Sizes and Projections
 
-Raw event data is streamed in individual batches that contain up to 100 events. Batches are posted at least several times every 30 seconds to 1 minute, even if the number of events is less than 100. (Batches may come from multiple servers, so the intervals of data posting may be irregular).
+Raw event data is streamed in individual batches that may vary from 1 to 1000 or more events based on your volume and peak sending rate. Batches are posted at least several times every 30 seconds to 1 minute. Batches may come from multiple servers, so the intervals of data posting may be irregular.
+
+This is how batch sizing/sending works:
+
+* If you have a batch that is at least 100 events, SparkPost transmits that as a single batch.
```
Comment (bizob2828) [inline articles/tech-resources/managing-webhook-data.md]: I wouldn't provide this level of detail. These batches mean nothing to the customer.  They can't control which MTA receives the email injection and how many events are occurring on each mta

Comment (sptratledge) [reply articles/tech-resources/managing-webhook-data.md]: Hey Bob - let's get some feedback from @kierancooper on that - I get your point, but you wouldn't believe the level of detail some of our customers want, and I've been asked SUPER detailed questions about how we do webhooks. Is there any downside to exposing this information to the customer, even if it's ultimately not useful to most?

Comment (bizob2828) [reply articles/tech-resources/managing-webhook-data.md]: Then we need to breakdown terminology.  I can try to take a stab at it. but as that reads right now it's not very accurate

Comment (sptratledge) [reply articles/tech-resources/managing-webhook-data.md]: Okay, sure - apologies for that. I understood what we were saying but the wording is quite difficult!

Comment (bizob2828) [reply articles/tech-resources/managing-webhook-data.md]: exactly.  I'll take a stab in the morning, I was just heading out.

**Pull request merged**